scipy/optimize/tests/test_minpack.py

```diff
@@ -517,6 +517,26 @@ def test_maxfev_and_bounds(self):
         assert_allclose(popt1, 2, atol=1e-14)
         assert_allclose(popt2, 2, atol=1e-14)
 
+    def test_curvefit_covariance(self):
```

Comment:
That was a bug. I have now fixed it. There should be no lower argument to cholesky or solve_triangular. Thanks for catching this.